### DISCUSSION TITLE
更新后打开502错误

### DISCUSSION THREAD

**Discussion by 520yxl** [AlistGo/alist#889] [Q&A]
使用curl -fsSL "https://nn.ci/alist.sh" | bash -s update 2.2.0更新2.3.1然后502了

**Comment by xhofe** [comment]
没启动。502是nginx错误

**Reply by 520yxl** [reply]
启动后，可以打开了，但是之前的数据连接不上了

**Reply by xhofe** [reply]
MySQL新增的参数：https://alist-doc.nn.ci/docs/setting/config#ssl_mode
